src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import React from "react";
+import s from "./BlockQuoteBlock.module.css";
+
+const BlockQuoteBlock = ({ quote, author }) => {
+  const GridAreaMiddle = () => (
+    <div className={s.gridAreaMiddle}>
+      <div className={s.quote}>{quote}</div>
+      <br />
+      <div className={s.author}>{author}</div>
+    </div>
```

Comment:
I was looking up how to semantically write this, since this is a rare opportunity to make use of the [`<blockquote>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote) tag, and I found [this article](https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/#hey-what-about-the-figure-element), which has a suggestion that I like. Basically, it suggests putting both the quote and the attribution in a `<figure>` element, the quote itself in a `<blockquote>` element, and the attribution in a `<figcaption>` element.

By the way, can we rename `author` to `attribution` here, since it can be more than the person's name, and also "author" to me implies someone who _wrote_ something, but "attribution" is a bit more general.

Reply:
Also, I think it'd probably be nicer to make the em dash part of the React component itself so that you don't have to place it in the `author` prop for every quote.

Reply:
But if I use the figure, blockquote, figcaption element, do I use the CSS too? If I do, it does not really look like the design on Figma. And if I don't include the CSS, what is the point of using these three elements?

Reply:
> But if I use the figure, blockquote, figcaption elements, do I use the CSS too?

Yes, you should still style the elements, since we don't want them to look like their default styles.

> If I do, it does not really look like the design on Figma.

Then that's your job to write the CSS such that it matches the designs.

> And if I don't include the CSS, what is the point of using these three elements?

You should write custom CSS for these elements, but the point of using them is to have more semantic HTML. [Here's a decent article](https://html.com/semantic-markup/) explaining what semantic HTML is and why we care about using it. To provide a brief summary here, using semantic tags allows you to more precisely describe the content of the page in HTML alone, regardless of how it is styled or rendered in a browser. These are important for navigating the HTML in non-visual ways, such as for search engine crawlers or screen readers, which are devices that allow blind or otherwise visually impaired users navigate the web.

Reply:
👍 Used semantic elements figure, blockquote, and figcaption for readability and clarity.